### ISSUE TITLE
why the createmask function is 49*49?

### ISSUE THREAD

**Issue by henbucuoshanghai** [berniwal/swin-transformer-pytorch#21]
def create_mask(window_size, displacement, upper_lower, left_right):
    mask = torch.zeros(window_size ** 2, window_size ** 2)

it is 49*49 in all tne swin network,why?

**Comment by henbucuoshanghai** [comment]
dots[:, :, -nw_w:] += self.upper_lower_mask
 dots[:, :, nw_w - 1::nw_w] += self.left_right_mask

why ?56*56**96  the first 3 line to 53,54 55 lines?
